GnuPlotSharp/Program.cs

```diff
@@ -82,40 +86,101 @@ public class GnuPlotScript
         public GnuPlotScript(string title)
         {
             this.title = title;
-        }        
+        }
 
-        public RenderResults Render<T1, T2>(params Row<T1, T2> [] rows)
+        public RenderResults Render<T1, T2>(params Row<T1, T2>[] rows)
         {
             var outputfile = Path.Combine(Path.GetTempFileName() + ".png");
 
-            return Render(outputfile, rows);
+            if (rows is Row<string, T2>[] _rows)
+                return Render(outputfile, _rows);
+            else
+                return Render(outputfile, rows);
         }
 
-        public RenderResults Render<T1, T2>(string outputfile, params Row<T1, T2> [] rows)
+        public RenderResults Render<T2>(string outputfile, params Row<string, T2>[] rows)
         {
-            var scriptFile = Path.GetTempFileName() + ".txt";            
+            Thread.CurrentThread.CurrentCulture = CultureInfo.GetCultureInfo("en");
```

Comment:
I think this will cause a side effect whereby it would be changing the culture of the calling code, any calling code would end up using the english culture after calling Render.
Instead it might be better to use something like this on the string interpolation functions:
```csharp
scriptContent += FormattableString.Invariant($@"plot [{xmin}:{xmax}] [{ymin}:{ymax}] ");
```

Reply:
It does indeed. But since the `System.FormattableString` class was introduced in .NET Framework 4.6, the minumum version for the project has to be raised again. I think that this should be no issue in most cases though.